jolie/src/jolie/runtime/typing/Type.java

```diff
@@ -73,7 +72,6 @@ public void cutChildrenFromValue( Value value )
 		}
 	}
 
-	@Override
 	protected Value cast( Value value, StringBuilder pathBuilder )
```

Comment:
The best way to implement this is probably to re-implement cast from TypePair, which could then call cast at the first type and then, if that fails, cast at the second type? The same goes for method check.